iothub/service/src/Registry/Models/ExportJobProperties.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains properties of an export job.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/rest/api/iothub/service/createimportexportjob"/>
+    public class ExportJobProperties
```

Comment:
These classes do share a fair number of properties. Consider making a base class they both inherit from to avoid the DRY violations.